README.MD

```diff
@@ -32,13 +32,22 @@ Currently [release-10](https://github.com/chain4travel/camino-messenger-protocol
 To run the camino-bot-messenger as intended you will need a **CM Account** with a **bot** added to it and **registered services**.
 
 ### Creating the CM Account
-If you don't already have a **CM Account**, for development purposes, a CM Account can be created via [camino messenger contracts](https://github.com/chain4travel/camino-messenger-contracts) `hardhat` tool or via [suite](https://suite.camino.network) as explained [here](https://docs.camino.network/partners/partner-config/#step-by-step-configuration-guide).
+If you don't already have a **CM Account** your company should [get listed as a Partner](https://docs.camino.network/partners/partner-showroom/index.html#being-listed-as-a-partner) and [configure its partner profile](https://docs.camino.network/partners/partner-config) via [suite](https://suite.camino.network). Following this process will result in a CM account and all requirements to run the bot on mainnet Camino. Creating the CM Account this way is necessary for the mainnet Camino.  
+
+For development purposes, it is also possible to configure the partner profile on testnet Columbus.
+
+For development purposes, a CM Account can also be created and configured without the company being listed as a Partner - via [camino messenger contracts](https://github.com/chain4travel/camino-messenger-contracts) `hardhat` tool.
 
 To create a **CM Account** and add the bot to it - you will need **two** (test) **wallets** - one for the **CM Account**, and one for the **bot**. The CM Account wallet has to be **KYC verified**. Creating and KYC verifying the test wallets can be done via [suite](https://suite.camino.network/create) as explained [here](https://docs.camino.network/guides/how-to-create-a-wallet) and [here](https://docs.camino.network/guides/kyc#kyc-verification-process).
 
 > [!IMPORTANT]
 > Remember to ensure the suite is connected to testnet Columbus if you are doing this for development purposes. 
 
+## Funding your wallet 
+
+> [!IMPORTANT]
+> Funds must be available on both wallets before you can create the CM account. At least 100 CAM (+ gas price and transaction fee) in the wallet used to create the CM Account and 100 CAM (+ gas price and transaction fee) on bot wallet. To request testnet funds, go to our [discord](https://discord.gg/XhrZTBXv) and use the command `/faucet <columbus-x-chain-address> <amount>` (max amount is 150CAM every 72 hours)
```

Comment:
```suggestion
> Funds must be available on both wallets before you can create the CM account. At least 100 CAM (+ gas price and transaction fee) in the wallet used to create the CM Account and 100 CAM (+ gas price and transaction fee) on bot wallet. To request testnet funds, go to our [discord](https://discord.gg/XhrZTBXv) and use the command `/faucet <columbus-x-chain-address> <amount>` (max amount is 150CAM every week)
```

Reply:
and we can also write that this is the current status and it may differ later, or we don't mention the maximum amount here because it will be shown by the discord bot

Reply:
@mo-c4t I added the link to documentation which should be updated to sad it may differ later
https://docs.camino.network/developer/guides/how-to-deploy-a-smart-contract#3-request-funds-from-the-discord-faucet